docs/stacks-guide-azure.md

```diff
@@ -0,0 +1,429 @@
+    namespace: azure-infra-dev
+  reclaimPolicy: Delete
+EOF
+
+kubectl apply -f environment.yaml
+```
+
+The steps that we have taken so far have been related to things that can be
+shared by all resources in all namespaces of that Crossplane cluster. Now, we
+will keep going with an app namespace which we will populate with resources that
+will help Crossplane know with what configuration it should satisfy the claims.
+You can use any namespace for your app's resources but for this tutorial we'll
+use the `app-project1-dev` namespace we created.
+
+Now we need to tell Crossplane which resource classes should be used to satisfy
+our claims in that app namespace. We will create portable classes that have have
+reference to non-portable ones that we created earlier. In our claims, we can
+refer to those portable classes directly or label one as the default portable
+class to be used in claims that do not have class reference.
+
+For example, MySQLInstanceClass is a portable class that can refer to Azure's
+SQLServerClass, which is a cloud-specific class.
+
+```
+cat > namespace.yaml <<EOF
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstanceClass
+metadata:
+  name: standard-mysql
+  namespace: app-project1-dev
+  labels:
+    default: "true"
+classRef:
+  kind: SQLServerClass
+  apiVersion: database.azure.crossplane.io/v1alpha2
+  name: azure-mysql-standard
+  namespace: azure-infra-dev
+---
+apiVersion: compute.crossplane.io/v1alpha1
+kind: KubernetesClusterClass
+metadata:
+  name: standard-cluster
+  namespace: app-project1-dev
+  labels:
+    default: "true"
+classRef:
+  kind: AKSClusterClass
+  apiVersion: compute.azure.crossplane.io/v1alpha2
+  name: azure-aks-standard
+  namespace: azure-infra-dev
+---
+EOF
+
+kubectl apply -f namespace.yaml
+```
+
+For more details about what is happening behind the scenes, read more about
+[portable claims in Crossplane][portable-claims].
+
+## Configure Network Connection
+
+After the Wordpress stack is installed, we will need the AKS Cluster it
+provisions to be able to communicate with the MySQL database it provisions. In
+Azure, we can do so using a [Virtual Network Rule][vnet-rule]. However, the rule
+cannot be created until after the MySQLInstance claim is created and satisfied,
+so we will start a short script to continually check if the database exists, and
+will create the rule if so.
+
+```bash
+cat > vnet-rule.yaml <<EOF
+apiVersion: database.azure.crossplane.io/v1alpha2
+kind: MysqlServerVirtualNetworkRule
+metadata:
+  name: demo-vnet-rule
+  namespace: azure-infra-dev
+spec:
+  name: demo-vnet-rule
+  serverName: MYSQL_NAME
+  resourceGroupName: demo-rg
+  properties:
+    virtualNetworkSubnetId: /subscriptions/${SUBSCRIPTION_ID}/resourceGroups/demo-rg/providers/Microsoft.Network/virtualNetworks/demo-vnet/subnets/demo-subnet
+  providerRef:
+    name: demo-azure
+    namespace: azure-infra-dev
+  reclaimPolicy: Delete
+EOF
+
+cat <<EOF > vnetwatch.sh
+#!/bin/bash
```

Comment:
Bash style: It's a nitpick in this case, but I would suggest that we use `#!/usr/bin/env bash` instead, just on general principle, so that we are making fewer assumptions about the user's environment. The `/usr/bin/env` form runs `bash` from wherever it lives in the environment, instead of assuming that it lives in a specific path.